packages/SwingSet/tools/manual-timer.js

```diff
@@ -53,7 +53,7 @@ const setup = () => {
  * kernel. You can make time pass by calling `advanceTo(when)`.
  *
  * @param {ManualTimerOptions} [options]
- * @returns {ManualTimer}
+ * @returns {TimerService & { advanceTo: (when: Timestamp) => void; }}
```

Comment:
is `ManualTimer` too hard to import here, or not correct, or what?

Reply:
It's not correct, due to the incomplete Timer migration. `ManualTimer` is now supposed to have `tick` and `tickN` instead of `advanceTo`.